keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -553,8 +558,8 @@ For Beta and GA, add links to added tests together with links to
 k8s-triage for those tests:
 https://storage.googleapis.com/k8s-triage/index.html -->
 
-For alpha, integration tests will be added to exercise the new
-aggregated discovery code path.
+Integration tests
+- test/integration/apiserver/discovery/discovery_test.go
```

Comment:
Nit: Those should be links not to code, but to testgrid, based on the [template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md). Same for the e2e section.